UX-Guide-Metadata/draft/techniques/onix-metadata/index.html

```diff
@@ -291,6 +291,198 @@ <h4>Instructions</h4>
+						<span><b>THEN</b> display <code id="conformance-claim">"This publication claims to meet "</code>.</span>
+					</li>
+					<li>
+						<span><b>IF</b> <var>epub_accessibility_1_0</var>:</span>
+						<span><b>THEN</b> display <code id="conformance-epub-accessibility-1-0">" EPUB Accessibility 1.0 "</code>.</span>
+					</li>
+					<li>
+						<span><b>ELSE IF</b> <var>epub_accessibility_1_1</var>:</span>
+						<span><b>THEN</b> display <code id="conformance-epub-accessibility-1-1">" EPUB Accessibility 1.1 "</code>.</span>
+					</li>
+					<li>
+						<span><b>IF</b> <var>wcag_2_2</var>:</span>
+						<span><b>THEN</b> display <code id="conformance-wcag-2-0">" WCAG 2.2 "</code>.</span>
+					</li>
+					<li>
+						<span><b>ELSE IF</b> <var>wcag_2_1</var>:</span>
+						<span><b>THEN</b> display <code id="conformance-wcag-2-1">" WCAG 2.1 "</code>.</span>
+					</li>
+					<li>
+						<span><b>ELSE IF</b> <var>wcag_2_0</var>:</span>
+						<span><b>THEN</b> display <code id="conformance-wcag-2-2">" WCAG 2.0 "</code>.</span>
+					</li>
+					<li>
+						<span><b>IF</b> <var>level_aaa</var>:</span>
+						<span><b>THEN</b> display <code id="conformance-level-aaa">" Level AAA"</code>.</span>
+					</li>
+					<li>
+						<span><b>ELSE IF</b> <var>level_aa</var>:</span>
+						<span><b>THEN</b> display <code id="conformance-level-aa">" Level AA"</code>.</span>
+					</li>
+					<li>
+						<span><b>ELSE IF</b> <var>level_a</var>:</span>
+						<span><b>THEN</b> display <code id="conformance-level-a">" Level A"</code>.</span>
+					</li>
+					<li>
+						<span><b>IF</b> <var>certification_date</var> <b>OR</b> <var>certifier</var> <b>OR</b> <var>certifier_credentials</var>:</span>
+						<span><b>THEN</b> display <code id="conformance-certification-info">"The publication was certified "</code>.</span>
+					</li>
+					<li>
+						<span><b>IF</b> <var>certification_date</var>:</span>
+						<span><b>THEN</b></span>
+						<ul class="condition">
+							<li>display <code id="conformance-ceertification-date-prefix">" on "</code></li>
+							<li>display <var>certification_date</var>.</li>
+						</ul>
+					</li>
+					<li>
+						<span><b>IF</b> <var>certifier</var>:</span>
+						<span><b>THEN</b></span>
+						<ul class="condition">
+							<li>display <code id="conformance-certifier-prefix">" by "</code></li>
+							<li>display <var>certifier</var>.</li>
+						</ul>
+					</li>
+					<li>
+						<span><b>IF</b> <var>certifier_credentials</var>:</span>
+						<span><b>THEN</b></span>
+						<ul class="condition">
+							<li>display <code id="conformance-certifier-credentials-prefix">" with a credential of "</code></li>
+							<li>display <var>certifier_credentials</var>.</li>
```

Comment:
Same here for live certification credential webpage.
However I guess technically this could be just the name of the certification not a URL so we would need to check if its a URL then make it live.

Reply:
Added, thanks